app/views/publish/courses/course_button_panel/_published.html.erb

```diff
@@ -3,6 +3,10 @@
 </span>
 
 <div class="govuk-button-group">
+  <% if FeatureService.enabled?(:open_and_closed_course_flow) %>
```

Comment:
I can't remember if this is supposed to be before the publish button or after. cc @simonwhatley ?